src/Client/ClientBase.cpp

```diff
@@ -2369,6 +2374,7 @@ void ClientBase::init(int argc, char ** argv)
         ("hardware-utilization", "print hardware utilization information in progress bar")
         ("print-profile-events", po::value(&profile_events.print)->zero_tokens(), "Printing ProfileEvents packets")
         ("profile-events-delay-ms", po::value<UInt64>()->default_value(profile_events.delay_ms), "Delay between printing `ProfileEvents` packets (-1 - print only totals, 0 - print every single packet)")
+        ("print-num-processed-rows", "print the number of locally processed rows")
```

Comment:
It feels to me that this name would be more elegant.
```suggestion
        ("processed-rows", "print the number of locally processed rows")
```